.github/workflows/build-images.yaml

```diff
@@ -46,6 +46,10 @@ jobs:
           image: ${{ env.OPERATOR_NAME }}
           tags: ${{ env.IMG_TAGS }}
           platforms: linux/amd64,linux/arm64
+          build-args: |
+            COMMIT=${{ github.sha }}
```

Comment:
like in limitador operator https://github.com/Kuadrant/limitador-operator/blob/main/.github/workflows/build-images-base.yaml#L78

let it be 
```
GIT_SHA=${{ github.sha }}
```